src/fu-engine.c

```diff
@@ -6452,6 +6453,10 @@ fu_engine_is_plugin_name_disabled(FuEngine *self, const gchar *name)
 		if (g_strcmp0(name_tmp, name) == 0)
 			return TRUE;
 	}
+
+	if (!fu_engine_config_get_test_mode(self->config) && g_str_has_prefix(name, "test"))
```

Comment:
this seems super fragile. Could we have those two plugins adding `FWUPD_REMOTE_FLAG_ONLY_TEST_MODE` in their `init()` and then checking for that instead?

Reply:
Yeah; I'll work out a new flag for them to use instead.

Reply:
fixed